### PR TITLE
Remove ISM changes from Wazuh indexer

### PR DESCRIPTION
**Pull request by rauldpm**
## Description

This pull request removes the ISM changes introduced in the Wazuh indexer package

## Logs + Tests example

See https://github.com/wazuh/wazuh-packages/issues/2834#issuecomment-1954975128

## Tests

- Build the package in any supported platform
  - [x] Linux
- [x] Package installation

- Tests for Linux RPM
  - [x] Build the package for x86_64
  - [x] `%files` section is correctly updated if necessary
- Tests for Linux deb
  - [x] Build the package for x86_64

### CODE REVIEW

**Review by juliamagan — CHANGES_REQUESTED** [review]
GJ! Just one change to make the code the same as before.

**Review by juliamagan — APPROVED** [review]
LGTM

File: stack/indexer/rpm/wazuh-indexer.spec [hunk 1]
```diff
@@ -93,7 +93,6 @@ cp %{REPO_DIR}/wazuh-passwords-tool.sh ${RPM_BUILD_ROOT}%{INSTALL_DIR}/plugins/o
 cp /root/documentation-templates/wazuh/config.yml ${RPM_BUILD_ROOT}%{INSTALL_DIR}/plugins/opensearch-security/tools/config.yml
 
 # Copy Wazuh's config files for the security plugin
-cp %{REPO_DIR}/config/indexer/roles/action_groups.yml ${RPM_BUILD_ROOT}%{CONFIG_DIR}/opensearch-security
```
Comment (AlexRuiz7) [inline stack/indexer/rpm/wazuh-indexer.spec:96]: Keep file or extend changes to roles, internal_users, roles_mappings and so on.

File: stack/indexer/deb/debian/rules [hunk 2]
```diff
@@ -100,7 +100,6 @@ override_dh_install:
 	cp /root/documentation-templates/wazuh/config.yml $(TARGET_DIR)$(INSTALLATION_DIR)/plugins/opensearch-security/tools/config.yml
 
 	# Copy Wazuh's config files for the security plugin
-	cp -pr $(REPO_DIR)/config/indexer/roles/action_groups.yml $(TARGET_DIR)$(CONFIG_DIR)/opensearch-security/
```
Comment (AlexRuiz7) [inline stack/indexer/deb/debian/rules:103]: Keep file or extend changes to roles, internal_users, roles_mappings and so on.

**Review by AlexRuiz7 — CHANGES_REQUESTED** [review]

File: stack/indexer/deb/debian/rules (hunk 2 above)
Comment (rauldpm) [reply stack/indexer/deb/debian/rules:103]: We should not maintain unused configuration, removed at https://github.com/wazuh/wazuh-packages/pull/2838/commits/3feaa8e3ccea8f36a1a5f33a8cee536d807633eb

**Review by juliamagan — APPROVED** [review]

**Pull request merged**